src/main/java/uk/gov/ida/matchingserviceadapter/MatchingServiceAdapterConfiguration.java

```diff
@@ -67,6 +67,10 @@
     @JsonProperty
     private EuropeanIdentityConfiguration europeanIdentity;
 
+    @Valid
+    @JsonProperty
+    private boolean shouldSignWithSHA1 = false;
```

Comment:
should this be a @NotNull property and be added to YAML files under `configuration`? Not sure how we typically deal with flags